src/Base58.mo

```diff
@@ -0,0 +1,65 @@
+import Text "mo:base/Text";
+import Char "mo:base/Char";
+import Nat8 "mo:base/Nat8";
+import Array "mo:base/Array";
+import Iter "mo:base/Iter";
+
+module {
+  // All alphanumeric characters except for "0", "I", "O", and "l".
+  private let pszBase58 : [Char] = [
```

Comment:
nit: I recommend not using abbreviations in variable names as it makes the code more difficult to read. As someone unfamiliar with base58 encoding, I don't know what "psz" stands for and would prefer to see the expanded name if needed.

Reply:
makes sense, changed.